include/cairo/context.hpp

```diff
@@ -126,6 +126,22 @@ namespace cairo {
       return *this;
     }
 
+    context& operator<<(const circle_segment& segment) {
```

Comment:
Can the `<<` operator for `rounded_corners` also reuse this code? It seems there is a lot of code in common here.

EDIT: Maybe not entirely, since this also does lines.

Reply:
I've removed the duplicate lines that were in there somehow and moved the degree value to a constant to avoid recalculating it every time the method is called.

I also don't think repurposing the `cairo_arc` method for use with `rounded_corners` AND `circle_segment` is necessary, but you tell me :D